pyiceberg/catalog/rest.py

```diff
@@ -237,6 +237,7 @@ def _create_session(self) -> Session:
         session.headers["Content-type"] = "application/json"
         session.headers["X-Client-Version"] = ICEBERG_REST_SPEC_VERSION
         session.headers["User-Agent"] = f"PyIceberg/{__version__}"
+        session.headers["X-Iceberg-Access-Delegation"] = "vended-credentials,remote-signing"
```

Comment:
I don't think we can send both as not all FileIO implementations support `remote-signing`.  We might just want to send vended credentials for now.

Reply:
indeed, I've updated this